docs/src/pages/docs/functions/helpers.md

```diff
@@ -662,6 +662,42 @@ safeMax(data)
 mconcat :: Monoid m, Foldable f => m -> f a -> m a
 ```
 
+
+```javascript
+import mconcat from 'crocks/helpers/mconcat'
+
+import Pred from 'crocks/Pred'
+import compose from 'crocks/helpers/compose'
+import getPropOr from 'crocks/helpers/getPropOr'
+import flip from 'crocks/combinators/flip'
+
+// Record that needs validation
+const person = { name: 'Maisie', age: 22, referral: 'ANON' }
+
+// Helpers -- predicate functions
```

Comment:
We use type signatures for comments above functions. check this out https://hackernoon.com/function-type-signatures-in-javascript-5c698c1e9801

This wouls be
```
// isOfAge :: Number -> Boolean
```